apps/11gist-blog/script.js

```diff
@@ -1,5 +1,136 @@
 'use strict';
 
 $(document).ready(function() {
-  // You code here
+    // Your code here
+
+    // make request to GET array of gist json objects
+    $.ajax('https://api.github.com/users/chrisperk/gists', {
+    // $.ajax('http://127.0.0.1:8080/apps/11gist-blog/api/gists.json', {
+
+        // if request is successful run this generate #posts list function with returned posts array as param
+        success: function(posts) {
+
+            // for every post object in the returned array run this generate #posts list-links function
+            posts.forEach(function(post) {
+
+                // check if a post object in the returned array's description prop starts with '#post' then continue if true
+                if (post.description.startsWith('#post')) {
+
+                    // get the #post description string, remove '#post ' from it, and store it in var
+                    var strPostDescription = post.description.slice(5);
```

Comment:
The comment explains nicely, but this is what we call a [magic number](http://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad)

Something to think about!